deployment/cfn/utils/cfn.py

```diff
@@ -42,30 +44,42 @@ def get_availability_zones(aws_profile):
     return [AvailabilityZone(az) for az in conn.get_all_zones()]
 
 
+def get_availability_zones(aws_profile, zones):
```

Comment:
This is an overloaded method to allow us to specify specific AZs objects we want to retrieve to assist in reconciling the VPC stack drift.